src/workerd/api/actor.h

```diff
@@ -322,6 +323,12 @@ class DurableObjectClass: public jsg::Object {
     // No methods - this is just a handle that gets passed to ctx.facets.get()
   }
 
+  void serialize(jsg::Lock& js, jsg::Serializer& serializer);
+  static jsg::Ref<DurableObjectClass> deserialize(
+      jsg::Lock& js, rpc::SerializationTag tag, jsg::Deserializer& deserializer);
```

Comment:
Nit: It occurs to me that we could probably just bundle the method decls into the `JSG_SERIALIZABLE` macro perhaps? Would we ever have `JSG_SERIALIZABLE` without these?